libcontainer/intelrdt/intelrdt.go

```diff
@@ -370,22 +371,21 @@ func WriteIntelRdtTasks(dir string, pid int) error {
 
 // Check if Intel RDT is enabled
 func IsIntelRdtEnabled() bool {
```

Comment:
This function should really just be called `IsEnabled` like `go-selinux` (the package name already tells us that it's `intelrdt`, there's no need to add it to the function name as well).

Reply:
@cyphar 

Thank you. I will correct the function name.